setup.py

```diff
@@ -169,7 +169,7 @@ def run(self):
         # Ensure our temporary build directory exists
         try:
             os.makedirs(build_temp)
-        except IOError:
+        except:
```

Comment:
Testing the directory exists has a race condition. What happens if the directory starts to exist between when you did the check if it exists and when you attempt to create it.

#140 should fix the thing where ``install`` builds stuff.